nibRealUI/Libs/Fixes/InspectFix.lua

```diff
@@ -9,7 +9,7 @@ local InspectGuildFrame_Update = nil
 local loaded = false
 local debugging = false
 local allowDetarget = false   	-- allow window to remain open when target is not inspectable (experimental)
-local serverTimeout = 1		-- timeout for INSPECT_READY response to assume server dropped it 
```

Comment:
Looks like my environment is set to normalize line endings automatically. Not sure if that's good, bad, or indifferent.